components/tooltip/tooltip.vue

```diff
@@ -21,7 +21,7 @@
     <dt-lazy-show
       :id="id"
       ref="content"
-      :show="isShown && (!!message.trim() || !!$slots.default)"
+      :show="isShown && !disabled && (!!message.trim() || !!$slots.default)"
```

Comment:
This definetly need a computed property which evaluate this condition...
The name of this computed property should be descriptable to explain what `!disabled && (!!message.trim() || !!$slots.default)` is doing...

Reply:
actually the whole thing can go in isVisible computed poperty ?

Reply:
Sounds good.

I would update how the `show` and `isShown` are being used, to work with the new `!disabled && (!!message.trim() || !!$slots.default)` evaluation.

Reply:
update as in documentation ?

Reply:
this `isVisible` will replace the `isShown` ?

Reply:
No isVisible is basically the entire condition. I probably did not get what you meant by replacing this ? As in just for readability ?

Reply:
Ye I mean... with this addition we will have `show`, `isShow` and `isVisible`... Im wondering whats going on...

Reply:
Update this PR with the `isVisible` approach, then I will take a look further on this, will try to come up with some solution.

Reply:
done but am not sure if thats required since the comments on each prop do mention what they do.

also to note, enabled prop is redundant in a way that if message is not provided it does the same thing. we introduced this prop so it will be explicit on that behaviour.